components/alttextify/actions/delete-image/delete-image.mjs

```diff
@@ -0,0 +1,26 @@
+import alttextify from "../../alttextify.app.mjs";
+
+export default {
+  key: "alttextify-delete-image",
+  name: "Delete Image Alt Text",
+  description: "Delete the generated alt text for a specific image using the asset ID. [See the documentation](https://apidoc.alttextify.net/#api-Image-DeleteImage)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    alttextify,
+    assetId: {
+      type: "string",
+      label: "Asset ID",
+      description: "The ID of the asset for retrieving or deleting alt text.",
+    },
```

Comment:
`assetId` should use async options and be in the app file. https://apidoc.alttextify.net/#api-Image-GetImages